catalog/admin/module/catalog-admin-module-sources/npm-shrinkwrap.json

```diff
@@ -1,3519 +0,0 @@
-{
```

Comment:
I removed them because I couldn't regenerate them with the updated versions and have them be compatible on windows. I new that they were getting removed anyway with the yarn update so I just removed them. I will remove them from this pr.